### ISSUE TITLE
EmailConverter.outlookMsgToEmail duplicates recipients

### ISSUE THREAD

**Issue by Faelean** [bbottema/simple-java-mail#241]
I'm currently using v6.0.0-rc2. 
When converting an Outlook .msg file to Email like this

```
//ByteArrayInputStream
Email email = EmailConverter.outlookMsgToEmail(stream));

List<Recipient> recipients = email.getRecipients()
```
the list of recipients contains every 'cc' entry also as a 'to' entry. For example if I have an Email like this
```
To: support@company.com
Cc: teamleadSupport@company.com
```
the list of recipients contains three entries,
```
[{
	address = 'support@company.com', 
	type = 'To'
},
{
	address = 'teamleadSupport@company.com', 
	type = 'To'
},
{
	address = 'teamleadSupport@company.com', 
	type = 'Cc'
}]
```
instead of the expected two.

**Comment by bbottema** [comment]
That's not good. Do you have a sample .msg for me to debug?

**Comment by Faelean** [comment]
I've send you an email containing a sample .msg.

**Comment by bbottema** [comment]
Bug is fixed, waiting for confirmation whether I can keep the provided test data in the code or if I should roll my own.

**Comment by bbottema** [comment]
Fixed in 6.0.0-rc3.

**Comment by bbottema** [comment]
6.0.0 has released as well, finally.
